report/2.Analysis/ArduinoInEducation.tex

```diff
@@ -0,0 +1,50 @@
+
+The electronics course focuses on engaging the students in making arudino projects, not in making the students understand the theory behind the projects. This means that the students both have a very limited understanding of the arudino platform's architecture and a low understanding of the arduino language explained earlier in the report, which they utilize in the electronics course on HTX CAO.
+
+Because of all this the electronics course have a relatively high learning curve. Which results in a high number of students abandoning the course in favor of some of the courses with a more specificly defined theory structures.
+
+Clearly there is a majojr gap between the students who are interested in programming or electronics and have no prior experience and students who have prior experiences. 
+Based on students' and the teacher's statements it seems like the course needs a better gateway into the arduino platform, the programming aspects of the course or both.
+
+A substantial number of the students in the class expressed their frustation with relative complexity of the arduino language. The teacher supplemented with saying that the language is not necessary complex, but the time spent on the different concepts are far too short, which is a result of the education being a junior education and not a college/university level education.
+Clearly there is a need for the course to be able to make shortcuts in some aspects to teach the students the concepts adequately. Put simply the language and the platform are not well suited for this kind of low level education.
+
+\subsubsection{The Students Understanding of Syntax and Semantics}
+Most of the students expressed that they have no real interest in the syntax. In fact a substantial part indicated that they did not understand what the syntax of a programming language actually means. Below is a short summarization of the most problematic aspects of general syntax and semantic of the arduino language (or any reference language the students might have), in arbitary order.
+
+\textbf{Data Types}\\
+Data types in general, seems to cause alot of problems for the students. Most students can name the different types but far less have the proper knowledge to identify and explain the meaning behind them. The students in most cases do not understand why they have to state the data type explicitly when declaring a variable. When presented with a small example program utilizing type inference on all declarations, the students in most cases, instantly understood how to identify a declaration and how to use the variable properly.
```

Comment:
Change "Data types in general, seems to cause alot of problems for the students." to "Data types in general, seem to cause problems for the students"